README.md

```diff
@@ -26,43 +26,52 @@ This README is the definitive source for downloading, installing, and running th
         sudo apt install python3-rosdep # if not already installed
         sudo rosdep init # if this is the first time using rosdep
 
-5. Install [`rosdep`](https://docs.ros.org/en/humble/Tutorials/Intermediate/Rosdep.html) dependencies:
+5. (Only for users **with** sudo access) Install [`rosdep`](https://docs.ros.org/en/humble/Tutorials/Intermediate/Rosdep.html) dependencies:
 
         rosdep update
         cd ~/colcon_ws
-        rosdep install --from-paths src -y --ignore-src
+        rosdep install --from-paths src -y --ignore-src --as-root=pip:false
 
-6. Install non-`rosdep` dependencies:
+6. (Only for users **without** sudo access) Install pip dependencies:
+
+        cd ~/colcon_ws/src/ada_feeding
+        python3 pip install -r requirements.txt
+
+7. Remove the duplicate matplotlib pip installation caused by installing scikit-spatial with pip: 
+
+        python3 pip uninstall matplotlib
```

Comment:
FWIW, on Sriram's account on `t0b1`, we got permission denied when we tried that, and had to run it in `sudo`. Perhaps worth documenting?